modules/client/src/main/java/org/apache/ignite/internal/client/compute/ClientCompute.java

```diff
@@ -215,9 +216,11 @@ private <T, R> CompletableFuture<SubmitResult> submit0(JobTarget target, JobDesc
             var mapper = (Mapper<? super Object>) colocatedTarget.keyMapper();
 
             if (mapper != null) {
-                return doExecuteColocatedAsync(colocatedTarget.tableName(), colocatedTarget.key(), mapper, descriptor, arg);
+                // need to be fixed after: https://issues.apache.org/jira/browse/IGNITE-24301 Fix Thin client protocol to use schema name
+                return doExecuteColocatedAsync(quoteIfNeeded(colocatedTarget.tableName()), colocatedTarget.key(), mapper, descriptor, arg);
```

Comment:
can`t use "quote" here cause this method processed recursively and appends additional quotation in such a case